don't install $bindir/{vi,ex,view,input} because you are a
replacement, not the official thing
Alejandro

--- ./Makefile.orig	2005-02-10 00:49:31.000000000 -0300
+++ ./Makefile	2005-02-10 00:50:18.178348278 -0300
@@ -349,10 +349,6 @@
 	cp $(PROGS) $(BIN)
 	(cd $(BIN); chmod 755 $(PROGS))
 	(cd $(BIN); chown bin $(PROGS))
-	-ln $(BIN)/elvis $(BIN)/vi
-	-ln $(BIN)/elvis $(BIN)/ex
-	-ln $(BIN)/elvis $(BIN)/view
-	-ln $(BIN)/elvis $(BIN)/input
 
 inst.dos: $(DUMMY)
 	copy $(PROGS) $(BIN)
